tsconfig.json

```diff
@@ -1,68 +1,28 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
   "compilerOptions": {
-    /* Basic Options */
```

Comment:
Cleaning this file up so that JSON LSPs can deal with them - comments are not valid JSON.